pkgs/top-level/all-packages.nix

```diff
@@ -28669,6 +28669,7 @@ with pkgs;
 
   alfis = callPackage ../applications/blockchains/alfis {
     inherit (darwin.apple_sdk.frameworks) Cocoa WebKit;
+    dialog = gnome.zenity;
```

Comment:
I don't understand why this is not named zenity.

Reply:
`alfis` supports not just `zenity`, but also `kdialog`, for example.

Reply:
Can you only compile against one or all?

It is not a good pattern to override packages to other names. dialog is also a package which gets shadowed by this and makes debugging unnecessarily hard.